src/clang_complete.cc

```diff
@@ -425,10 +428,12 @@ void CompletionQueryMain(ClangCompleteManager* completion_manager) {
 
             ls_completion_item.documentation = ToString(
                 clang_getCompletionBriefComment(result.CompletionString));
-            ls_completion_item.sortText =
-                (const char)uint64_t(GetCompletionPriority(
+
+            buf << std::hex << std::setw(16) << std::setfill('0') <<
+                uint64_t(GetCompletionPriority(
```

Comment:
Remove uint64_t cast?

Reply:
The return type of `GetCompletionPriority` may need changing.